source/compiler/CMakeLists.txt

```diff
@@ -69,6 +69,14 @@ endif()
 if(APPLE)
   set(CMAKE_MACOSX_RPATH ON)
 endif()
+
+if(DEFINED PAWN_MAX_FUNC_NAME
```

Comment:
OK, `sNAMEMAX` is not a great name, but it is known.  It would be nice to just keep that name rather than a new more complex one.